plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/project/ProjectUtils.java

```diff
@@ -127,6 +127,7 @@
     progress.setWorkRemaining(10 * projectFiles.size() + 10);
     List<IProject> projects = new ArrayList<>(projectFiles.size());
     for (IPath projectFile : projectFiles) {
+      System.out.println("Importing project from: " + projectFile);
```

Comment:
any reason to use a logger here instead of System.out? For that matter, does this need to be checked in or can you read the results from this PR alone?